plugins/package-managers/pub/src/main/kotlin/model/Pubspec.kt

```diff
@@ -71,7 +72,7 @@ internal data class Pubspec(
     /** See https://dart.dev/tools/pub/dependencies#hosted-packages. */
     @Serializable
     data class HostedDependency(
-        val version: String,
+        val version: String?,
```

Comment:
For consistency with `url`, shouldn't this also have a default value of `null`?

Note that if this class would be used with regular deserialization, and not with a custom serializer only, an explicit default value of `null` would even be required, as KxS in contrast to Jackson does not assign `null` to nullable types that are missing from the input by default.

Reply:
I've made another PR: https://github.com/oss-review-toolkit/ort/pull/9313